packages/react-network/src/server.ts

```diff
@@ -29,7 +29,10 @@ export function applyToContext<T extends Context>(
     // only set cookies that have been changed
     // decode URI as the manager's cookie value is also decoded
     const rawCookieValue = ctx.cookies.get(cookie);
-    if (rawCookieValue && decodeURIComponent(rawCookieValue) === value) {
+    if (
+      typeof rawCookieValue !== 'undefined' &&
```

Comment:
this will let false & null pass through, is that the intent?

Reply:
I guess the type for rawCookieValue can only be string or undefined...

Reply:
Yeah exactly 😄